src/components/Cluster/ClusterDetail/ClusterDetailView.js

```diff
@@ -296,6 +310,16 @@ class ClusterDetailView extends React.Component {
                   onSave={this.editClusterName}
                 />{' '}
               </h1>
+              {(isClusterUpdating(cluster) || isClusterCreating(cluster)) && (
+                <SlideTransition in={true} appear={true} direction='down'>
+                  <NotReadyNotice>
+                    <i className='fa fa-info' /> This cluster is currently{' '}
+                    {isClusterCreating(cluster) ? 'creating' : 'updating'}.
```

Comment:
Can we store the `isClusterCreating(cluster)` value into a variable? There's no need to compute it twice